phys2bids/tests/test_txt.py

```diff
@@ -26,7 +26,6 @@ def loaded_lab_file(multifreq_lab_file):
     header_lab, channels_lab = txt.read_header_and_channels(multifreq_lab_file, chtrig)
 
     # just a few quick checks to make sure the data loaded correctly
```

Comment:
```suggestion
    # just a few quick checks to make sure the data loaded correctly
    assert len(channels_lab[ 0]) == 5
```

Reply:
I updated the file with a comment in the first line of the channels. This should trigger it. Can you do accept this change so we can see if it works?